client/app/lib/components/membershiprolechangedmodal/index.coffee

```diff
@@ -0,0 +1,29 @@
+articlize = require 'indefinite-article'
+ContentModal = require 'app/components/contentModal'
+
+
+module.exports = class MembershipRoleChangedModal extends ContentModal
+
+  constructor: ({ role, adminNick }) ->
+
+    options =
+      title : 'Your team role has been changed!'
+      overlay : yes
+      width : 500
+      cssClass : 'content-modal'
+      content :
+        """
+        <p>
+          @#{adminNick} made you #{articlize role} <strong font-weight="bold">#{role}</strong>,
+          please refresh your browser for changes to take effect.
+        </p>
+        """
+      buttons :
+        'Reload page' :
+          title : 'Reload Page'
+          style : 'solid medium'
+          callback : (event) =>
+            @destroy()
```

Comment:
this works but it's not looking good to referencing `this` before `super` on line 29. just wanted to mention.